docs/tutorials/custom_images.md

```diff
@@ -8,7 +8,7 @@ By default, Fae includes a thumbnail size called `thumb` that is triggered like
 = image_tag(item.image.asset.thumb.url)
 ```
 
-You may copy the contents of the `uploaders/fae/image_upload.rb` file and add your own options, such as mobile and tablet and call them the same way.
+You may copy the contents of the `uploaders/fae/image_uploader.rb` file and add your own options, such as mobile and tablet and call them the same way. Don't forget to also add `gem 'rmagick'` to youre gemfile and then run bundle install.
```

Comment:
The Rmagick reference was due to this file being out of date, I recently updated it. Fae now dependeds on `mini_magick` and is already listed as a gem dependency (https://github.com/wearefine/fae/blob/master/fae.gemspec#L30)

Remove the sentence about Rmagick and I'll give it a merge 👍

Reply:
Removed it